xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-docker/src/main/java/org/xwiki/test/docker/junit5/TestConfiguration.java

```diff
@@ -459,4 +461,13 @@ public String getOutputDirectory()
         }
         return outputDirectory;
     }
+
+    /**
+     * @return the path of the directory containing the test resources on the container.
```

Comment:
I guess you mean "in the container", right?

Reply:
wait, what container? :) This missing.

Reply:
I don't think this is the right place. You should put this in the Browser enum probably (similar to what we do for the other enums - See what you removed from the ServletEngine enum).